### PR TITLE
P1 components

### PR DESCRIPTION
**Pull request by Muzammil-khan710**

### CODE REVIEW

File: docs/components/cards/cards.css [hunk 1]
```diff
@@ -43,6 +43,7 @@
     top: 2px;
     right: 5px;
     color: var(--color-red);
+    padding: 5px;
```
Comment (palash23jain) [inline docs/components/cards/cards.css:46]: you could have this in relative unit like rem or em. But not a breaking change as of now

**Review by palash23jain — APPROVED** [review]
good to go

File: docs/components/cards/cards.css (hunk 1 above)
Comment (Muzammil-khan710) [reply docs/components/cards/cards.css:46]: values are very small so i am using px here. Thank you for the suggestion

**Review by Shubhamkr2610 — APPROVED** [review]
Code is clean . Every thing looks good to me. You can merge it.

**Pull request merged**